apps/chrome/chrome.py

```diff
@@ -38,9 +39,27 @@ def tab_close_wrapper():
         actions.sleep("180ms")
         actions.app.tab_close()
 
+def chrome_app():
+    return ui.apps(bundle="com.google.Chrome")[0]
 
 @ctx.action_class("browser")
 class browser_actions:
+    def address() -> str:
+        try:
+            window = chrome_app().windows()[0]
```

Comment:
I believe you can simplify this to `ui.active_app.windows()[0]`, no?

Reply:
Because this is triggered a lot (on every window/tab/title change) I was worried it might trigger after the current frontmost/active app changed so I wanted to address the browser directly.

Reply:
It’s not a big deal, but I don’t think that’s a concern here since the Chrome/Safari app has to be active for this implementation of address() to be active at all. I believe the browser scope is evaluated sometime later.

Reply:
I was worried about a race condition if the user changes apps while the script is running. But maybe I am being overly cautious.

Reply:
Fair enough, works for me.